backend_new/types/src/backend-swagger.ts

```diff
@@ -455,6 +455,135 @@ export class ReservedCommunityTypesService {
   }
 }
 
+export class UnitTypesService {
+  /**
+   * List unitTypes
+   */
+  list(options: IRequestOptions = {}): Promise<UnitType[]> {
+    return new Promise((resolve, reject) => {
+      let url = basePath + '/unitTypes';
+
+      const configs: IRequestConfig = getConfigs(
+        'get',
+        'application/json',
+        url,
+        options,
+      );
+
+      /** 适配ios13，get请求不允许带body */
```

Comment:
Do you know why non-english words and characters are here?

Reply:
I'm fairly certain that it found its way in the nestjs/swagger package, hopefully at some point they fix that though

Reply:
So there isn't anything we can do right now? Is it worthwhile to manually change it even though it will get overwritten next time it is run?

Reply:
so with each new model that gets built out we would need to update

I feel like maybe at the end when permissions are applied we can go through and clean that up

I may also look to create a PR for that package to clean that up when I get a chance

Reply:
@YazeedLoonat , can you make sure an issue is filed for this first. Could already be taken care of.